src/HX711_ADC.h

```diff
@@ -11,7 +11,7 @@
 
 #include <Arduino.h>
 
-#define SAMPLES 			16									// no of samples in moving average data set, value must be 4, 8, 16, 32 or 64
+#define SAMPLES 			32									// no of samples in moving average data set, value must be 4, 8, 16, 32 or 64
```

Comment:
I'm not sure that this is a good idea, in 10sps mode a reading will take >3 seconds?